Fix listings.sty in some circumstance
Fix by Qing Lee on TL mailing list 2014-06-28
---
 texmf-dist/tex/latex/listings/listings.sty |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

--- texlive-base-2014.20140626.orig/texmf-dist/tex/latex/listings/listings.sty
+++ texlive-base-2014.20140626/texmf-dist/tex/latex/listings/listings.sty
@@ -1888,10 +1888,11 @@
       \def\lst@boxpos{b}%
       \lsthk@PreSet\lstset{flexiblecolumns,#1}%
       \lsthk@TextStyle
+      \ifnum\iffalse{\fi`}=\z@\fi
       \@ifnextchar\bgroup{%
-        \ifnum`{=0}\fi%
-        \afterassignment\lst@InlineG \let\@let@token}%
-                         \lstinline@}
+        \ifnum`{=\z@}\fi%
+        \afterassignment\lst@InlineG \let\@let@token}{%
+        \ifnum`{=\z@}\fi\lstinline@}}
 \def\lstinline@#1{%
     \lst@Init\relax
     \lst@IfNextCharActive{\lst@InlineM#1}{\lst@InlineJ#1}}
